src/cluster/k8sClientHandler.go

```diff
@@ -498,3 +502,106 @@ func GetKubearmorRelayURL() string {
 	url := "kubearmor." + namespace + ".svc.cluster.local"
 	return url
 }
+
+// GetSecrets to get secrets
+func GetSecrets(k8sClient *kubernetes.Clientset, label string) (*v1.Secret, error) {
+	if k8sClient == nil {
+		return nil, errors.New("k8s client not created")
+	}
+
+	secrets, err := k8sClient.CoreV1().Secrets("").List(context.Background(), metav1.ListOptions{
```

Comment:
Since we are creating this secret under 'kube-system' namespace with a predefined name can't we use `Secrets.Get` to get the secret?